README.md

```diff
@@ -2,23 +2,16 @@
 
 ## Getting started
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
+The project uses vagrant to run in a suitably configured VM. You will need Vagrant and VirtualBox installed.
```

Comment:
Good job switching over to poetry fully and deleting requirements.txt.  You just missed out deleting `setup.sh`

And what you've done is fine, but I don't think the app is so complex yet that you need to force people to use vagrant. You could just add a line saying to install python & poetry and then run `poetry install` & `poetry run flask run`. Especially if they still need that to run the tests